src/apps/allbridge/api.ts

```diff
@@ -0,0 +1,93 @@
+import got from 'got'
+import { NetworkId } from '../../types/networkId'
+import { Address } from 'viem'
+
+type SupportedAllbridgeChainSymbols =
+  | 'ETH'
+  | 'CEL'
+  | 'POL'
+  | 'ARB'
+  | 'OPT'
+  | 'BAS'
+
+const NETWORK_ID_TO_ALLBRIDGE_BLOCKCHAIN_SYMBOL: Record<
+  NetworkId,
+  SupportedAllbridgeChainSymbols
+> = {
+  [NetworkId['ethereum-mainnet']]: 'ETH',
+  [NetworkId['ethereum-sepolia']]: 'ETH',
```

Comment:
the allbridge info is mainnet only right? is it ok if we map testnet to mainnet info as well? Wonder if there would be issues when getting positions for testnets